src/core/io/src/4C_io_value_parser.hpp

```diff
@@ -247,7 +248,20 @@ namespace Core::IO
       }
     }
 
-    template <typename U, typename... SizeInfo>
+    template <typename... Ts>
+    void read_internal(std::tuple<Ts...>& value)
+    {
+      std::apply([this](auto&... val) { (read_internal(val), ...); }, value);
```

Comment:
@lauraengelhardt I think you run into [this bug](https://github.com/llvm/llvm-project/issues/36228) with clang. Try:


```suggestion
      std::apply([this](auto&... val) { (this->read_internal(val), ...); }, value);
```